resources/sdk/purecloudjava/templates/ApiClient.mustache

```diff
@@ -560,15 +567,18 @@ public class ApiClient implements AutoCloseable {
         ApiClientConnectorRequest connectorRequest = prepareConnectorRequest(request, isAuthRequest);
         ApiClientConnectorResponse connectorResponse = null;
         try {
-            connectorResponse = connector.invoke(connectorRequest);
+            Retry retry = new Retry(retryConfiguration == null ? new RetryConfiguration() : retryConfiguration);
```

Comment:
By default this will new RetryConfiguration() for every request. It might be worth creating a static final default RetryConfiguration and using that instead of creating a new one for each request.